Source/WebCore/dom/Subscriber.cpp

```diff
@@ -148,6 +151,35 @@ void Subscriber::reportErrorObject(JSC::JSValue value)
     reportException(globalObject, JSC::Exception::create(vm, value));
 }
 
+Vector<VoidCallback*> Subscriber::teardownCallbacksConcurrently()
+{
+    Locker locker { m_teardownsLock };
+    return m_teardowns.map([](auto& callback) {
+        return callback.ptr();
+    });
+}
+
+InternalObserver* Subscriber::observerConcurrently()
+{
+    return &m_observer.get();
+}
```

Comment:
What’s the rationale for moving this out of the header? Might be a good idea, but the change log doesn’t seem to say why.